src/plugins/query_enhancements/public/query_assist/components/query_assist_summary.tsx

```diff
@@ -185,6 +185,7 @@ export const QueryAssistSummary: React.FC<QueryAssistSummaryProps> = (props) =>
             total_count: queryContext?.queryResults?.length,
             question: queryContext?.question,
             ppl: queryContext?.query,
+            index: selectedDataset.current?.title,
```

Comment:
Do we need to check if dataset is index type first? For index pattern dataset seems we will pass the index pattern name as index. Is it an expected behavior?